templates/web/pages/bmviii_home_temp.tmpl

```diff
@@ -0,0 +1,265 @@
+{{ define "bmviii home temp" }}
+{{ template "head" . }}
+
+{{ template "bmviii navbar" . }}
+
+{{ template "flashes" .Flashes }}
+{{ template "mlh badge" }}
+
+<section class="section has-background-image" style="background-color: #A7C3FF; position: relative; padding-top: 0px; padding-bottom: 0px; padding-left: 0px; padding-right: 0px">
+    <img alt="BM buildings" src={{ static_path "bmviii_lp_buildings.svg" }} style="vertical-align: bottom">
+    <div class="container bmviii-front-logo logo-div" style="top: 7%;">
+        <img class="logo-img" alt="Boilermake" src={{ static_path "bmviii_boilermake_logo.svg" }}>
+    </div>
+    <div class="container bmviii-front-logo" style="top: 23%; padding-left: 2.5%">
+        <img alt="Lay back and hack" src={{ static_path "bmviii_lay_back_and_hack.svg" }} width="80%">
+    </div>
+    <div class="container bmviii-front-logo" style="top: 37%; padding-left: 5%">
+        <a href="https://forms.gle/ugmBaA4bHvm6rwjF7">
+            <img alt="Apply Now!" src={{ static_path "bmviii_apply_now.svg" }} width="65%">
+        </a>
+    </div>
+    <div class="container bmviii-front-logo" style="top: 60%; padding-left: 2.3%">
```

Comment:
Using super exact percentage values like this usually means that the layout your making is gonna struggle to deal with screen widths that are different than the screen with you're working with when developing.  For example, on a phone the page isn't readable:
![Screen Shot 2020-10-05 at 21 04 52](https://user-images.githubusercontent.com/10781907/95147327-0a7cfa80-0746-11eb-8935-a114142b1d48.png)

Reply:
Do you have any ideas on how to resolve this? Using a media query and have it increase in percentage is the first thing that comes to mind for me

Reply:
columns can help again here, if you say it should take up like 8/12 columns on desktop then it'll do that just fine, and then on mobile it'll take up the full width (and you can add some padding to one side of the other if you still want to keep that bubble-thing aesthetic).

Reply:
Which bubble?

Reply:
oh sorry I was calling the whole covid-19 blurb a bubble for some reason

Reply:
Oh gotchu, I used columns for that part. I don't think it was included in this pr tho

Reply:
With columns, it's also kinda hard to control the spacing between each text horizontally.